### PR TITLE
Implement bench backrest quest

### PR DESCRIPTION
**Pull request by Map3428**
Fix https://github.com/westnordost/StreetComplete/issues/484

A neat icon is still missing, I can't design it.

### CODE REVIEW

**Comment by ENT8R** [comment]
> A neat icon is still missing, I can't design it.
I can try to create one.

**Comment by matkoniecz** [comment]
Also, what about cases where both picnic table and bench are mapped? User may mistakenly retag benches to tables.

BTW, is there any, even potential use for this data?

**Comment by westnordost** [comment]
Perhaps useful information for the physically impaired / old people? @rugk seemed to be enthusiastic about this quest.

**Comment by rugk** [comment]
So as for the use case, it may not only be for visually impared people. I still remember that OsmAnd may display difficult icons or so. 😄 

Also there is even a map for that: https://www.mapcontrib.xyz/t/4da70b-Bancs_et_tables_de_pique_nique#

> Also, what about cases where both picnic table and bench are mapped?

AFAIK this is not a valid mapping is it? Otherwise one may just need to remove the picnic option?

> (Is missing an icon. Suggestion: A walking cane.)

No, please not. Not only can this potentially be confused with tactile paving quests and similar, but it does also not really indicate that this is about a bench.
It is way easier: Just use a bench as an icon for that. AFAIK there is not a other quest, which may be asked about benches, so the icon would not even be used twice, potentially. (and even if it would, that is not so bad, as it the icon is used in the toilets quests or so twice)

**Comment by westnordost** [comment]
I added a bench icon to the repository 🎉

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -463,4 +463,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_internet_access_no">No connection</string>
     <string name="quest_busStopShelter_tram_name_title">"Does the tram stop \"%s\" have a shelter?"</string>
     <string name="quest_busStopShelter_tram_title">"Does this tram stop have a shelter?"</string>
+    <string name="quest_bench_backrest_title">Does this bench has a backrest?</string>
```
Comment (rugk) [inline app/src/main/res/values/strings.xml]: *have

**Comment by westnordost** [comment]
Okay I did this now. Congrats for your first quest, @Map3428 :-)

**Pull request merged**